frontend/src/exporter/Exporter.tsx

```diff
@@ -100,6 +123,29 @@ export function Exporter(props: ExportedData): JSX.Element {
                     noBorder={exportedData.noBorder ?? false}
                     accessToken={exportToken}
                 />
+            ) : type === ExportType.Heatmap ? (
+                <div className="flex justify-center h-screen w-screen overflow-scroll heatmap-exporter relative">
+                    <HeatmapCanvas
+                        positioning="absolute"
+                        widthOverride={null}
+                        context="in-app"
+                        exportToken={exportToken}
+                    />
+                    <iframe
+                        id="heatmap-iframe"
+                        ref={null}
+                        className="h-screen bg-white w-screen"
+                        // eslint-disable-next-line react/forbid-dom-props
+                        src={exportedData.heatmap_url ?? ''}
+                        onLoad={() => {}}
+                        // these two sandbox values are necessary so that the site and toolbar can run
+                        // this is a very loose sandbox,
+                        // but we specify it so that at least other capabilities are denied
+                        sandbox="allow-scripts allow-same-origin"
```

Comment:
oooh, did i ever get around to removing the dependency on the toolbar here? if so we can remove the allowscripts sandbox

(i don't remember, and not blocking here!)